Library/Formula/gdb.rb

```diff
@@ -24,13 +24,12 @@ class Gdb < Formula
     sha256 "1c48758042eed1ba9357fdb98b49b5b819555d664f9bb876e35c8bbaad58b0ec" => :mavericks
   end
 
-  option "with-brewed-python", "Use the Homebrew version of Python"
+  option "with-python", "Use the Homebrew version of Python; by default system Python is used"
```

Comment:
@mikemcquaid Did you mean something like the following?
```
  deprecated_option "with-brewed-python" => "with-python"

  option "with-python", "Use the Homebrew version of Python; by default system Python is used"
```
Also, a good description for this option is needed so not to confuse the end users. Could you please suggest any other if the current description is not easy to understand. I'll update this PR accordingly.